gossip/src/crds_value.rs

```diff
@@ -211,17 +211,17 @@ impl AccountsHashes {
     }
 }
 
-pub type SnapshotHashes = AccountsHashes;
+pub type LegacySnapshotHashes = AccountsHashes;
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, AbiExample)]
-pub struct IncrementalSnapshotHashes {
+pub struct LegacyIncrementalSnapshotHashes {
     pub from: Pubkey,
     pub base: (Slot, Hash),
     pub hashes: Vec<(Slot, Hash)>,
```

Comment:
Looks like this already includes `base` which I am guessing is the `full` snapshot hash;
So why do we need a new `SnapshotHashes`?

Reply:
Why we can't just use current `IncrementalSnapshotHashes` without any changes?
I mean https://github.com/solana-labs/solana/issues/30759 claims that:
> Bootstrap runs into issues when it is discovering snapshots due to full snapshots and incremental snapshots gossiping out separately. 

But from `IncrementalSnapshotHashes` struct, it does look like it is gossip both full and incremental snapshots together!

Reply:
> I would like to make a few naming changes though:
> 
>     1. rename `SnapshotHashes` and `IncrementalSnapshotHashes` to `LegacySnapshotHashes` and `SnapshotHashes`, respectively
> 
>     2. rename `base` and `hashes` to `full` and `incremental`, respectively

yeah, this sounds fine.


> I think it'd also be useful to reduce the value for `MAX_INCREMENTAL_SNAPSHOT_HASHES`, but that can be handled later.

Any reason to do this other than saving bytes from bandwidth?